src/database/config/database.js

```diff
@@ -7,11 +7,21 @@ config();
 
 const env = envConfigs[process.env.NODE_ENV] || envConfigs.development;
 const dbase = new Sequelize(env.url, {
-  dialectOptions: {
-    ssl: {
-      require: true,
-      rejectUnauthorized: false,
-    },
+  host: 'localhost',
+  dialect: 'postgres',
+  pool: {
+    max: 5,
+    min: 0,
+    acquire: 30000,
+    idle: 10000,
   },
 });
+// const dbase = new Sequelize(env.url, {
+//   dialectOptions: {
+//     ssl: {
+//       require: true,
+//       rejectUnauthorized: false,
+//     },
+//   },
+// });
```

Comment:
why these changes ???? 
The DB was already configured well.
Remove your changes here unless they add any value